packages/apps/webconnector/src/services/webConnector.ts

```diff
@@ -97,32 +103,32 @@ export default class webConnectorService extends TransactionBaseService {
 
       await this.oauthService_.update(oauth.id, { last_sync: new Date() });
 
-      const data = {
-        link: base_url,
-        link_id,
-        emit_event: false,
-        status: "success",
-      };
-      await this.organisationService.updateInstalledApp(
-        AppNameDefinitions.WEBCONNECTOR,
-        data
-      );
+      // const data = {
+      //   link_id,
+      //   emit_event: false,
+      //   status: "success",
+      //   org_id,
+      // };
+      // await this.organisationService_.updateInstalledApp(
+      //   AppNameDefinitions.WEBCONNECTOR,
+      //   data
+      // );
 
       this.logger_.info(
         `Finished oculation of Web Connector for ${org.id} organisation`
       );
     } catch (error) {
       console.error("Error fetching web commector content:", error);
-      const data = {
-        link: base_url,
-        link_id,
-        emit_event: false,
-        status: "failed",
-      };
-      await this.organisationService.updateInstalledApp(
-        AppNameDefinitions.WEBCONNECTOR,
-        data
-      );
+      // const data = {
```

Comment:
nit: remove this code block

Reply:
Which code block ?

Reply:
The commented code which is not being used.